tcms/kiwi_auth/forms.py

```diff
@@ -113,3 +115,26 @@ def save(  # pylint: disable=too-many-arguments
             html_email_template_name,
             extra_email_context,
         )
+
+
+class GroupAdminForm(forms.ModelForm):
+    class Meta:
+        model = Group
+        fields = ['name', 'permissions']
+
+    users = forms.ModelMultipleChoiceField(
+         queryset=User.objects.all(),
+         required=False,
+         widget=FilteredSelectMultiple('users', False)
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.pk:
+            self.fields['users'].initial = self.instance.user_set.all()
+
+    def save(self, commit=True):
+        instance = super().save(commit)
```

Comment:
`commit=commit` because it is a keyword argument, not a positional one.

Reply:
In this method I should pass commit=False in order to work properly. I have to save() the instance, call self.instance.user_set and then call save_m2m which cannot be done if commit=True. The current value that is passed to commit variable is False.